doc/source/multivariate.rst

```diff
@@ -144,12 +144,12 @@ We realize that the mean vector and the covariance often have special forms in p
 
 .. code-block:: julia
 
-    immutable MvNormal{Cov<:AbstractPDMat,Mean<:Union{Vector{Float64},ZeroVector{Float64}}} <: AbstractMvNormal
+    immutable MvNormal{Cov<:AbstractPDMat,Mean<:Union{Vector,ZeroVector}} <: AbstractMvNormal
```

Comment:
That took more work than I thought, but see the latest commit. Also fixed GenericMvTDist and MvNormalCanon so that eltypes are the same everywhere and eltype is the first type parameter.